packages/sdk/src/router/v6/router.ts

```diff
@@ -1462,6 +1452,69 @@ export class Router {
       }
     }
 
+    // Handle Rarible listings
+    if (raribleDetails.length) {
+      const orders = raribleDetails.map((d) => d.order as Sdk.Rarible.Order);
+      const module = this.contracts.raribleModule.address;
+
+      const fees = getFees(raribleDetails);
+
+      const totalPrice = orders
+        .map((order) => bn(order.params.take.value))
+        .reduce((a, b) => a.add(b), bn(0));
+      const totalFees = fees
+        .map(({ amount }) => bn(amount))
+        .reduce((a, b) => a.add(b), bn(0));
+
+      const order = orders[0];
```

Comment:
This variable seems redundant, since we use `orders[0]` for the single-order case.